### PR TITLE
vulnerability disclosure policy

### PR DESCRIPTION
**Pull request by lisac-usds**
## summary of changes
Introduces page `/vulnerability-disclosure-policy`

connects to https://github.com/usds/website/issues/582

screencap:

### CODE REVIEW

**Comment by boris-ning-usds** [comment]
Ooo, I like the formatting! Since I copied most of this from the [DHS VDP template](https://cyber.dhs.gov/bod/20-01/vdp-template/), I have no objections with it - so I'll defer to the other reviewers.

Thank you Lisa.

**Review by boris-ning-usds — DISMISSED** [review]

File: _content/vulnerability_disclosure.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ 
+    Though we develop and maintain other internet-accessible systems or services, we ask that active research and testing only be conducted on the systems and services covered by the scope of this document. If there is a particular system not in scope that you think merits testing, please contact us to discuss it first. We will increase the scope of this policy over time.
+
+reporting:
+  title: 'Reporting a vulnerability'
+  section_text: |-
+    Information submitted under this policy will be used for defensive purposes only – to mitigate or remediate vulnerabilities. If your findings include newly discovered vulnerabilities that affect all users of a product or service and not solely United States Digital Service, we may share your report with the other organizations as needed to coordinate and remediate the vulnerability. We will not share your name or contact information without express permission.
+
+    We accept vulnerability reports via <infrastructure@usds.gov>. Reports may be submitted anonymously. If you share contact information, we will acknowledge receipt of your report within 3 business days.
```
Comment (wslack) [inline _content/vulnerability_disclosure.yml]: Do we want to address if we do/do not support PGP e-mails?

Comment (lisac-usds) [reply _content/vulnerability_disclosure.yml]: addressed in 4a279f9. 
screencap of affected section:

File: _content/vulnerability_disclosure.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    We do not support PGP-encrypted emails. If you are concerned about the confidentiality of your email, ensure your Mail Transfer Agent uses SMTP over TLS.
```
Comment (alexdotgov) [inline _content/vulnerability_disclosure.yml]: 👏

**Review by alexdotgov — DISMISSED** [review]
I don't really understand the value of the multiple-sections in a template + yaml file vs. just a single markdown file, but the content LGTM.

**Comment by lisac-usds** [comment]
How this is broken up across the 2 files - that was driven by limited experience with the codebase! I’ll push an update later today that eliminates the .yml, which will make this new page resemble how  [contact-us.html](https://github.com/usds/website/blob/master/pages/contact-us.html) is wired up.

**Comment by nacingov** [comment]
I would strongly, strongly suggest adding security@ as an alias for infrastructure@. This serves both to use the industry-standard security@ and to not broadcast the infrastructure@ address.

**Pull request merged**